### ISSUE TITLE
Descriptions are poor if the contain an image

### ISSUE THREAD

**Issue by eaburns** [velour/feedme#171]
See: http://backcomic.com/rss.xml
We should probably grab the alt text. Unfortunately, backcomic doesn't have any alt text! In which case, we should probably just fallback on using the < link >, which, in this case, would look fine.
